src/main/java/org/cryptomator/cryptofs/event/ConflictResolvedEvent.java

```diff
@@ -0,0 +1,15 @@
+package org.cryptomator.cryptofs.event;
+
+import java.nio.file.Path;
+
+/**
+ * Emitted, if a conflict inside an encrypted directory was resolved
+ *
+ * @param cleartextPath path within the cryptographic filesystem
+ * @param ciphertextPath path to the encrypted resource
+ * @param oldVersionCleartextPath path within the cryptographic filesystem of the renamed resource
+ * @param oldVersionCiphertextPath path to the renamed, encrypted resource
+ */
+public record ConflictResolvedEvent(Path cleartextPath, Path ciphertextPath, Path oldVersionCleartextPath, Path oldVersionCiphertextPath) implements FilesystemEvent {
```

Comment:
as said before, I'd just use `oldCiphertextPath`, `newCiphertextPath` and `cleartextPath`

Reply:
also: Should this the canonical cleartext path? Or the on of the renamed resource?

Reply:
Changed in https://github.com/cryptomator/cryptofs/pull/277/commits/2cd07fae1fc779bfa798ba287919ba9cb69227af